theme/styles/global.scss

```diff
@@ -0,0 +1,13 @@
+.button--style-primary {
+  @extend %transition-shorter;
+  border-radius: 0.75rem;
+  border: 1px solid color('black');
+  text-transform: uppercase;
+  color: color('black');
+  text-decoration: none;
+
+  &:hover {
+    background-color: color('black');
+    color: color('white');
+  }
+}
```

Comment:
This is a global button style. Normally I would put this in `button.scss` for the `Button` component but I wasn't sure if it was necessary to create a `Button` snippet + `button.scss`!

Reply:
i'm not sure we need a button snippet, but I think this makes sense in a `buttons.scss` file instead of the `global.scss` file

Reply:
Cooool, thanks, will make a buttons.scss :)